### PR TITLE
Add --by-develproject argument to adi command

### PR DESCRIPTION
**Pull request by nilxam**
If with --by-develproject, adi command will check devel project of source package before selecting requests to adi staging.

### CODE REVIEW

File: osc-staging.py [hunk 1]
```diff
@@ -172,4 +174,9 @@ def do_staging(self, subcmd, opts, *args):
         elif cmd == 'list':
             ListCommand(api).perform()
         elif cmd == 'adi':
-            AdiCommand(api).perform(args[1:], opts.move)
+            if opts.move:
+                AdiCommand(api).perform(args[2:], opts.move)
+            elif opts.by_develproject:
+                AdiCommand(api).perform(args[2:], by_dp=opts.by_develproject)
+            else:
```
Comment (coolo) [inline osc-staging.py]: I wonder why you move from args[1:] to args[2:] - and I think you can just have move=opts.move, by_dp=opts.by_develproject in one line without ifs

Comment (nilxam) [inline osc-staging.py]: if with --move argument then args[1] is "--move", doesn't it? if use args[1:] ie. packages actually it's including "--move"?

Comment (coolo) [inline osc-staging.py]: args only contains the arguments not filtered out, i.e. --move moved into opts.move and out of args. same with by_develproject

Comment (nilxam) [inline osc-staging.py]: ok, so I misunderstand args and opts here, updated then.

File: osclib/adi_command.py [hunk 2]
```diff
@@ -38,7 +39,15 @@ def check_adi_projects(self):
         for p in self.api.get_adi_projects():
             self.check_adi_project(p)
 
-    def create_new_adi(self, wanted_requests):
+    def get_devel_project(self, project, package):
+        m = show_package_meta(self.apiurl, project, package)
```
Comment (coolo) [inline osclib/adi_command.py]: self has no apiurl, self.api.apiurl is the proper form

File: osclib/adi_command.py [hunk 3]
```diff
@@ -38,7 +39,15 @@ def check_adi_projects(self):
...
+        node = ET.fromstring(''.join(m)).find('devel')
```
Comment (coolo) [inline osclib/adi_command.py]: ET is not imported in the adi command

**Comment by coolo** [comment]
beside those 2 show stoppers it seems to work properly

**Pull request merged**